Xamarin.Forms.Platform.iOS/Forms.cs

```diff
@@ -209,7 +209,7 @@ public static void Init()
 
 		public static event EventHandler<ViewInitializedEventArgs> ViewInitialized;
 
-		internal static void SendViewInitialized(this VisualElement self, TNativeView nativeView)
+		public static void SendViewInitialized(this VisualElement self, TNativeView nativeView)
```

Comment:
Do you need this for something , this should be need for anything right now

Reply:
I am not sure if this is needed for the Xamarin Community Toolkit. In my original Popup Control that I PRed to Xamarin.Forms I followed similar techniques from other built-in renderers which used this.  Is this only to be used internal? If so I am fine leaving it internal.

Here are the links to the original PR fork that used this API
https://github.com/ahoefling/Xamarin.Forms/blob/489c149260ce116ac8c68705c45d511e29285c97/Xamarin.Forms.Platform.iOS/Renderers/PopupRenderer.cs#L64

```c#
Element?.SendViewInitialized(NativeView);
```

Reply:
Yo don't need to use this. it's not need anymore on the renderers. This was used to set th AutmoationID for UITests.

Reply:
That makes sense, thanks for the clarification